src/node/performance/worker_service_impl.js

```diff
@@ -36,6 +36,11 @@
 var BenchmarkClient = require('./benchmark_client');
 var BenchmarkServer = require('./benchmark_server');
 
+exports.quitWorker = function quitWorker(call, callback) {
+  callback(null, {});
+  process.exit(0);
```

Comment:
Yes, I think that's the intention. The driver should only invoke this after its previous test has completed so there shouldn't be calls hanging around.